new-site/src/docs/elements/components/checkbox/customisation.mdx

```diff
@@ -60,6 +60,19 @@ You can use the `style` property to customise the component. See all available t
 }}
 ```
 
+```html
+<div
+  class="hds-checkbox"
```

Comment:
The core editor throws an error: `SyntaxError: Invalid or unexpected token`
I guess we need to something for the editor 🤔

Reply:
Was able to fix this with applying html-validate library for core live error